### PR TITLE
Upated community contributions

### PR DESCRIPTION
**Pull request by kumar-kalyan**
## Fixes Issue
Added new contributions to profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1724"></a>

### CODE REVIEW

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Reusable

File: public/data/kumar-kalyan.json [hunk 1]
```diff
@@ -46,6 +46,13 @@
       "icon": "book",
       "color": "red",
       "description": "Started leading the technical communities as a student leader"
+    },
+    {
+      "title": "Became Google DSC Lead",
+      "date": "August 2022",
+      "icon": "book",
+      "color": "red",
+      "description": "As a chapter lead I am helping everyone to grow their knowledge in a peer-to-peer learning environment and build solutions for local businesses and communites"
```
Comment (Panquesito7) [inline public/data/kumar-kalyan.json:55]: Minor spelling error in here.
```suggestion
      "description": "As a chapter lead I am helping everyone to grow their knowledge in a peer-to-peer learning environment and build solutions for local businesses and communities"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by maharshi-sinha — APPROVED** [review]
Great, thanks for contributing🚀

File: public/data/kumar-kalyan.json (hunk 1 above)
Comment (eddiejaoude) [reply public/data/kumar-kalyan.json:55]: Good spot 👍 . I think if the author doesn't respond after 1 week, we should merge their PR, as other changes are more important for their profile.

They can submit a new PR to fix the typo

**Pull request merged**